rundeckapp/grails-app/assets/javascripts/knockout.debug.js

```diff
@@ -480,7 +480,7 @@ ko.utils = (function () {
                 else
                     throw new Error("The supplied element doesn't support dispatchEvent");
             } else if (useClickWorkaround && element.click) {
-                element.click();
+                element.on('click',);
```

Comment:
seems like it should be `.trigger('click')`?

Reply:
it depends click() is native now